librubyfmt/src/format_prism.rs

```diff
@@ -1727,6 +1728,7 @@ fn call_chain_elements_are_user_multilined(
             prism::Node::CallNode { .. }
                 | prism::Node::ConstantReadNode { .. }
                 | prism::Node::ConstantPathNode { .. }
+                | prism::Node::ParenthesesNode { .. }
```

Comment:
I'm not sure if this is accidental or intended in the Ripper implementation, but we check for leading literals (like arrays etc.) by [checking for specific Expression types](https://github.com/fables-tales/rubyfmt/blob/b2b28df470fd918b0ac667829d301a7d1032151a/librubyfmt/src/render_targets.rs#L384-L386), but in the Ripper call chain node types, [`Paren` is separate from `Expression`](https://github.com/fables-tales/rubyfmt/blob/b2b28df470fd918b0ac667829d301a7d1032151a/librubyfmt/src/ripper_tree_types.rs#L1719-L1720), so it is not considered a "leading expression".